### DISCUSSION TITLE
Button for auto-create random alias on email input fields

### DISCUSSION THREAD

**Discussion by ghost** [anonaddy/anonaddy#168] [Feature Requests]
SimpleLogin has a feature/add-on for Firefox, which lets you auto-create an alias with a button next to email input fields. This is a huge enhancement in the process and I think anonaddy could be improved by this. 
Love to see it! :)

**Comment by willbrowningme** [comment]
Thanks for the suggestion, I'll look into this and see if it's something I can add, I think it might require extra permissions for the extension if I remember correctly.

**Reply by pbl987** [reply]
Firefox Relay has a feature like this. If it's open source, you can reuse the code...
It's a bit buggy, by the way.

![ffrelay](https://user-images.githubusercontent.com/28102804/128195728-97a11418-6cc7-4026-9195-7c6aa5a4fb58.jpg)
![ffrelay2](https://user-images.githubusercontent.com/28102804/128195732-234eeab2-6a35-4818-be6d-1fa2590285d6.jpg)

**Reply by ghost** [reply]
> Firefox Relay has a feature like this. If it's open source, you can reuse the code...
> It's a bit buggy, by the way.
> 


SimpleLogin as well! Shouldn't be an issue, if @willbrowningme someday decides to spend his time with this :D

**Comment by eliliam** [comment]
Would love to shift from SimpleLogin to AnonAddy, but this feature is a must have, otherwise it's too cumbersome to use naturally for creating accounts on different sites.

**Reply by willbrowningme** [reply]
I'm still planning to add this feature. You can currently use the browser extension and set it to show the create alias page on opening, then it's just one click from there to create and automatically copy the alias.

So it's 2 clicks in total, so just 1 extra mouse click than the feature above.

**Reply by pbl987** [reply]
The Firefox extension is rather bad: it reloads all the time, when you close it, when you click a button, when you go back. Even with fast internet this is a hassle.
